recipe/meta.yaml

```diff
@@ -27,16 +26,16 @@ requirements:
     - pip
     - python
     - cython
+    - blas * netlib  # [unix]
```

Comment:
Does this mean the reference netlib BLAS? Why can't we use OpebBLAS as was the case before (at least on Linux)?

Reply:
Packages are built against netlib BLAS, but any implementation can be used at install time. See https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=blas#blas